ci/install-cloud-sdk.sh

```diff
@@ -26,7 +26,11 @@ ARCH="$(uname -m)"
 if [[ "${ARCH}" == "aarch64" ]]; then
   # The tarball uses this name
   ARCH="arm"
+else
+  # For x86_64, uname -p might give unknown
+  ARCH="$(uname -m)"
```

Comment:
Did this sneak back in again?

Reply:
Yes :(

I think I figured out how git works and have successfully removed the old commit after rebasing 20 times